examples/graph_examples/graph_from_text_example.py

```diff
@@ -0,0 +1,51 @@
+"""
+Example of custom graph using existing nodes
+"""
+
+import os
+from dotenv import load_dotenv
+from scrapegraphai.models import OpenAI
+from scrapegraphai.graphs import BaseGraph
+from scrapegraphai.nodes import TextNode, ParseHTMLNode, GenerateAnswerNode
+
+load_dotenv()
+
+# Define the configuration for the language model
+openai_key = os.getenv("OPENAI_APIKEY")
+llm_config = {
+    "api_key": openai_key,
+    "model_name": "gpt-3.5-turbo",
+    "temperature": 0,
+    "streaming": True
+}
+model = OpenAI(llm_config)
+
+text = open("text_example.txt", "r", encoding="utf-8")
+
+# define the nodes for the graph
+fetch_html_node = TextNode("fetch_html")
```

Comment:
It isn't supposed to read only html but general text, I would change to fetch_text_node